modules/sdk-coin-xlm/src/xlm.ts

```diff
@@ -67,10 +67,12 @@ interface RecoveryOptions extends InitiateRecoveryOptions {
 }
 
 interface RecoveryTransaction {
-  tx: string;
+  txBase64: string;
```

Comment:
Breaking change?

Reply:
Yes, please update the commit accordingly

Reply:
Done